code/__defines/armour.dm

```diff
@@ -0,0 +1,47 @@
+// Armor will turn attacks into less dangerous (e.g. turning cut into bruise), so keep that in mind when decided what armor value to use.
+// Some levels are marked with what they intend to block in such way.
+
+#define  ARMOR_BALLISTIC_MINOR		10
```

Comment:
I left the other values as I don't believe we have a universal value set we use for armor, and changing every piece of armor to fit the schema in an a PR for an event species felt out-of-scope to me, since they probably don't fit neatly

Reply:
Armor defines slain